modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/porosity_jump/formloss/tests

```diff
@@ -0,0 +1,24 @@
+[Tests]
+  issues = '#30184'
+  design = 'navier_stokes/pinsfv.md BernoulliPressureVariable.md'
+
+  [single-expansion]
+    type = CSVDiff
+    input = 'formloss_single.i'
+    csvdiff = 'formloss_single_out.csv'
+    requirement = 'The system shall be able to handle irreversible pressure drops from expansions at interfaces.'
```

Comment:
Do you have the analytical solution for this case? If so let s put it in the comments

Reply:
Yes, @rfkile prepared an excel sheet where he compared the results with his analytic expressions. The results were in a good agreement (0.1% was the worst). That's why I golded it with the output result.

Reply:
I will add a few comments here and there.